spec/integration/application_view/part_namespace_spec.rb

```diff
@@ -95,43 +104,37 @@ module CustomParts
   end
 
   context "view in application" do
-    let(:view_class) {
-      module TestApp
-        class View < Hanami::View
-        end
-      end
-
-      TestApp::View
-    }
+    let(:view_class) { TestApp::View::Base }
 
     context "parts_path configured" do
       let(:application_hook) {
         proc do
-          config.views.parts_path = "views/custom_parts"
+          config.views.parts_path = "view/custom_parts"
         end
       }
 
       context "namespace exists" do
         before do
           module TestApp
-            module Views
+            module View
               module CustomParts
               end
             end
           end
         end
 
-        it "is the matching module within the slice" do
-          is_expected.to eq TestApp::Views::CustomParts
+        # FIXME: @jodosha ask @timriley
```

Comment:
@timriley This is failing and IDK why. Could you please help me? Thanks 🙏

Reply:
@jodosha This was a load order issue with the tests (the view class was defined in the top-level `before` block, which ran before the context-level `before` block just above, which meant that it couldn't see the `CustomParts` module, since it wasn't defined yet.

Have pushed up a fix :)

Reply:
Thank you 🙏